tests/test_orient_symmetric.py

```diff
@@ -218,10 +227,13 @@ def testRelativeViewingDirections(self, order):
         angular_dist_viis = np.mean(min_theta_vii)
 
         # Check that estimates are indeed approximately rank 1.
-        # ie. check that the two smaller singular values are close to zero.
-        tol = 5e-2
-        self.assertTrue(np.max(sij[:, 1:]) < tol)
-        self.assertTrue(np.max(sii[:, 1:]) < tol)
+        # ie. check that the svd is close to [1, 0, 0].
+        error_ij = np.linalg.norm(np.array([1, 0, 0], dtype=self.dtype) - sij, axis=1)
+        error_ii = np.linalg.norm(np.array([1, 0, 0], dtype=self.dtype) - sii, axis=1)
+        self.assertTrue(np.max(error_ij) < 0.2)
```

Comment:
I see what you mean here.

The svd is working well

```
(Pdb) np.argsort(sij[:,0])[:10]
array([137, 369, 164, 203, 372,  55, 365, 128, 470, 366])
(Pdb) sij[137]
array([0.88687116, 0.05656426, 0.05656405])
(Pdb) vijs[137]
array([[-0.04872306, -0.29703991, -0.04824536],
       [ 0.29001243,  0.76238387,  0.12343965],
       [-0.04202397, -0.10931137, -0.07560748]])
(Pdb) (uij_est[137] * sij[137]) @ v[137]
array([[-0.04872306, -0.29703991, -0.04824536],
       [ 0.29001243,  0.76238387,  0.12343965],
       [-0.04202397, -0.10931137, -0.07560748]])
```

Traced it back to `vijs`

```
(Pdb) uij_gt, _, _ = np.linalg.svd(vijs_gt)
(Pdb) uij_gt[137]
array([[-0.33958263, -0.93900139,  0.05440607],
       [ 0.928971  , -0.3257727 ,  0.17574138],
       [-0.14729739,  0.11022038,  0.98293181]])
(Pdb) uij_est[137]
array([[-0.3384581 , -0.86799665, -0.36335647],
       [ 0.92990571, -0.3676036 ,  0.01195678],
       [-0.14394959, -0.33384039,  0.93157346]])
(Pdb) vijs_gt[137]
array([[-0.11297516, -0.31524119,  0.05635564],
       [ 0.30905774,  0.86238194, -0.15416795],
       [-0.04900411, -0.13673905,  0.02444483]])
(Pdb) vijs[137]
array([[-0.04872306, -0.29703991, -0.04824536],
       [ 0.29001243,  0.76238387,  0.12343965],
       [-0.04202397, -0.10931137, -0.07560748]])
```

I tried to trace it farther to see if I could sniff where the worst estimate was coming from but if I change this line alll the tests fail... and I think that might be a bug?

```
index 0f9fc690..b0d367bc 100644
--- a/tests/test_orient_symmetric.py
+++ b/tests/test_orient_symmetric.py
@@ -54,7 +54,7 @@ class OrientSymmTestCase(TestCase):
             self.cl_orient_ests[order] = CLSymmetryC3C4(
                 self.srcs[order],
                 symmetry=f"C{order}",
-                n_theta=self.n_theta,
+                n_theta=self.n_theta+1,
```

Reply:
Ok, I think there's a couple things going on here. It seems there is a bug here:https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/fad4dfd6ebf66d4983688c5ec01d99fae7167130/src/aspire/basis/polar_2d.py#L63-L64
There should be parentheses around `n_theta // 2`. But I think ultimately we should be restricting `self.n_theta` to only even numbers (which would make the above mentioned parentheses unnecessary). Otherwise we'll be omitting a ray whenever we take the polar Fourier with `n_theta` odd. 

There is this check in `commonline_base.py`: https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/fad4dfd6ebf66d4983688c5ec01d99fae7167130/src/aspire/abinitio/commonline_base.py#L72-L75

But it needs to be moved up a few lines above `evaluate_t()`.

I will make an issue covering this.